x-pack/plugins/observability/public/pages/slo_details/components/header_control.tsx

```diff
@@ -202,6 +203,7 @@ export function HeaderControl({ isLoading, slo }: Props) {
                 <EuiContextMenuItem
                   key="exploreInApm"
                   icon="bullseye"
+                  disabled={!hasApmReadCapabilities}
```

Comment:
would be nice to show tooltip around on why button is disabled.

Reply:
Yep I could, it would look like this. Does the text look good to you?

<img width="303" alt="Screenshot 2023-11-28 at 13 48 18" src="https://github.com/elastic/kibana/assets/2852703/c229b721-9608-467e-9d89-92e682379b02">

My only concern is that we don't show tooltips to the rest disabled icons. But this one I guess is different from the rest, cause it's the only one related to apm privileges and not SLO privileges.

@maciejforcone What do you think? Shall I show the tooltip in this case?

Reply:
I was thinking of disabling the entire "Actions" button if there are no available actions for the user (with limited permissions, like in this scenario).

Reply:
@maciejforcone Good idea! 'll merge current PR as it is to have the bug fixed and I will disable the Actions button in another PR with a few tests I want to cover.